client/my-sites/themes/theme-collections-layout.tsx

```diff
@@ -0,0 +1,76 @@
+import { memo, ReactElement } from 'react';
+import QueryThemes from 'calypso/components/data/query-themes';
+import ThemeCollection from 'calypso/components/theme-collection';
+import ThemeCollectionItem from 'calypso/components/theme-collection/theme-collection-item';
+import { ThemeBlock } from 'calypso/components/themes-list';
+import { THEME_COLLECTIONS } from 'calypso/my-sites/themes/collection-definitions';
+import { ThemesQuery, useThemeCollection } from 'calypso/my-sites/themes/use-theme-collection';
+
+interface ThemesCollectionsLayoutProps {
+	getActionLabel: ( themeId: string ) => string;
+	getOptions: ( themeId: string ) => void;
+	getScreenshotUrl: ( themeId: string ) => string;
+}
+
+function ThemeCollectionsLayout( {
+	getActionLabel,
+	getOptions,
+	getScreenshotUrl,
+}: ThemesCollectionsLayoutProps ) {
+	const ShowcaseThemeCollection = ( {
+		collectionSlug,
+		title,
+		description,
+		query,
+	}: {
+		collectionSlug: string;
+		title: string;
+		description: ReactElement;
+		query: ThemesQuery;
+	} ): ReactElement => {
```

Comment:
Let's turn these types into a `ShowcaseThemeCollectionProps` interface for the sake of my eyes. 😆